README.md

```diff
@@ -241,22 +241,22 @@ We use [bunyan](https://github.com/trentm/node-bunyan) for logging:
 * By default logging is off (level is set to 'fatal') .
 * Logging in tests can be enabled using an env var e.g: `LOG_LEVEL=debug grunt test`
 * Logging on the cli can be enabled with `--log-level [level]`.
-* Bunyan by default logs JSON. If you want the json to be pretty printed
+* Bunyan by default logs JSON. If you want the JSON to be pretty printed
   pipe anything that logs into `bunyan` e.g. `LOG_LEVEL=debug grunt test
   | node_modules/bunyan/bin/bunyan`
 
 
 ## Architecture
 
-In a nutshell the way the linter works is to take an add-on
-package, extract the metadata from the xpi (zip) format and then
+In a nutshell, the way the linter works is to take an add-on
+the package, extract the metadata from the xpi (zip) format and then
```

Comment:
This reads odd.

Reply:
Yeah, that `the` feels wrong, the comma should be fine though.

Reply:
Yep.